certified-connectors/Easyvista_Service._Manager/apiProperties.json

```diff
@@ -0,0 +1,54 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "host": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Easyvista Host",
+          "displayName": "Host",
+          "tooltip": "Enter Hostname"
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": "true",
+            "tabIndex": 3
+          },
+          "description": "EasyVista Password",
+          "displayName": "Password",
+          "tooltip": "Enter password"
+        }
+      },
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": true,
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "EasyVista Login",
+          "displayName": "Login",
+          "tooltip": "Enter Login"
+        }
+      }
+    },
+    "iconBrandColor": "#007d82",
+    "policyTemplateInstances": [
+      {
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('Host')"
+        },
+        "templateId": "dynamichosturl",
+        "title": "Dynamic Host URL"
+      }
+    ]
+  }
```

Comment:
@ezv-compteintegration Please add the "publisher" and "stackowner" properties.
```
"publisher": "Name of company or organization.",
"stackOwner": "Name of the owner of the service or product."
```
Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).